mlrun/projects/pipelines.py

```diff
@@ -220,9 +219,6 @@ def is_run_local(self, local=None):
         force_run_local = mlrun.mlconf.force_run_local
         if force_run_local is None or force_run_local == "auto":
             force_run_local = not mlrun.mlconf.is_api_running_on_k8s()
-            kfp_url = mlrun.mlconf.resolve_kfp_url()
-            if not kfp_url:
-                force_run_local = True
```

Comment:
This check is still needed and important, just use `mlrun.mlconf.kfp_url` instead of `mlrun.mlconf.resolve_kfp_url`.
For example running CE on k8s in docker desktop on Apple Silicon forces the state of k8s but no kfp.